<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qtairmon-ng</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="devicesTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Interface</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Driver</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Chipset</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>MAC address</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Monitor Mode</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="randMACButton">
       <property name="text">
        <string>Randomize all MAC addresses</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="allmonitorcheckBox">
       <property name="text">
        <string>Enable Monitor Mode on All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="refreshButton">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string>Kill interfering processes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="interfaceButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Select Interface</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
